### PR TITLE
`Assessment`: Show disabled correction rounds in assessment overview

### PR DESCRIPTION
**Pull request by JohannesStoehr**
#### General

- [x] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).
#### Client
- [x] I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client/) and ensured that the layout is responsive.
- [x] I added multiple screenshots/screencasts of my UI changes.
- [x] I translated all newly inserted strings into English and German.

### Motivation and Context

Some testers and tutors were confused about the second assessment round not showing up in the assessment page.

### Description

The assessment page now shows all assessment rounds that will be made during the assessment.
To reduce the overall size of the UI, the explanation about not being able to assess the same submission in different assessment rounds is moved to a tool tip.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student
- 1 Exam with a second assessment round

1. Log in to Artemis
2. Participate in the exam as the student
3. Go to the assessment board after the deadline of the exam and check the UI of the assessment page for the different exercises
4. Enable/Disable the second correction round and check for confusing UI

#### Code Review
- [x] Review 1
- [x] Review 2
#### Manual Tests
- [ ] Test 1
- [ ] Test 2

### Screenshots

**Old:**
Second Round disabled:
Bildschirmfoto 2022-07-28 um 19 08 52
Second Round enabled:
Bildschirmfoto 2022-07-28 um 19 09 04

**New:**
Second Round disabled:
Bildschirmfoto 2022-07-28 um 19 10 39
Second Round enabled:
Bildschirmfoto 2022-07-28 um 20 11 45

### CODE REVIEW

File: src/main/webapp/i18n/en/exerciseAssessmentDashboard.json [hunk 1]
```diff
@@ -101,6 +102,7 @@
             "noSubmissionsInfo": "You may not assess any submissions this round. {{notYetAssessed}} submissions have yet to be assessed, of which {{lockedSubmissionsByOtherTutor}} are currently locked by other tutors ",
             "noSubmissionsInfoSecondCorrection": " and {{firstRoundAssessments}} were assessed by you in the first round.",
             "noSubmissionsBecauseFirstAssessedByYourself": "In the second correction round you may only assess submissions which you did not assess in the first round.",
+            "correctionRoundNotYetEnabled": "This correctoion round is not yet enabled.",
```
Comment (ge65cer) [inline src/main/webapp/i18n/en/exerciseAssessmentDashboard.json:105]:
```suggestion
            "correctionRoundNotYetEnabled": "This correction round is not yet enabled.",
```

**Review by ge65cer — CHANGES_REQUESTED** [review]

**Review by Dominik-Weinzierl — APPROVED** [review]
Code changes look good to me - you could think about replacing some `!` with more defensive `?`

**Review by ge65cer — APPROVED** [review]
Code changes look reasonable.

**Review by akesfeden — APPROVED** [review]
Tested on TS2, works as described, the info message about second correction round is more clear now.

**Review by manuelmanso — APPROVED** [review]
LGTM

**Review by GeneralMine — APPROVED** [review]

**Pull request merged**